src/opencl_rawsha256_fmt_plug.c

```diff
@@ -607,9 +607,10 @@ static int crypt_all(int *pcount, struct db_salt *_salt)
 {
 	const int count = *pcount;
 	const struct db_salt *salt = _salt;
-	size_t gws = count;
-	size_t *lws = (local_work_size && !(gws % local_work_size)) ?
-		&local_work_size : NULL;
+	size_t gws, initial = 128;
+	size_t *lws = local_work_size ? &local_work_size : &initial;
+
+	gws = GET_MULTIPLE_OR_BIGGER(count, local_work_size);
```

Comment:
It seems I picked a bad name. I will propose a patch to rename it do `GET_NEXT_MULTIPLE`.